tests/parser/features/test_logging.py

```diff
@@ -156,12 +156,35 @@ def foo():
     assert c.translator.decode_event(logs.topics, logs.data) == {'arg1': [1, 2], 'arg2': [timestamp, timestamp + 1, timestamp + 2], 'arg3': [[1, 2], [1, 2]], '_event_type': b'MyLog'}
 
 
+def test_logging_with_input_bytes():
+    loggy_code = """
+MyLog: __log__({arg1: indexed(bytes <= 4), arg2: indexed(bytes <= 29), arg3: bytes<=31})
+
+def foo(arg1: bytes <= 29, arg2: bytes <= 31):
+    log.MyLog('bar', arg1, arg2)
+"""
+    c = get_contract(loggy_code)
+    c.foo('bar', 'foo')
+    logs = s.head_state.receipts[-1].logs[-1]
+    event_id = u.bytes_to_int(u.sha3(bytes('MyLog(bytes4,bytes29,bytes31)', 'utf-8')))
+    # # Event id is always the first topic
+    assert logs.topics[0] == event_id
+    # # Event id is calculated correctly
+    assert c.translator.event_data[event_id]
+    # # Event abi is created correctly
+    assert c.translator.event_data[event_id] == {'types': ['bytes4', 'bytes29', 'bytes31'], 'name': 'MyLog', 'names': ['arg1', 'arg2', 'arg3'], 'indexed': [True, True, False], 'anonymous': False}
+    # Event is decoded correctly
+    assert c.translator.decode_event(logs.topics, logs.data) ==  {'arg1': b'bar\x00', 'arg2': b'bar\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00', 'arg3': b'foo\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00', '_event_type': b'MyLog'}
```

Comment:
Is there some sort of helper we can give when working with bytes? Like `b'bar\x00'` I really only care that the first 3 is `bar` and the remaining entries are asserted to be empty. Maybe something like `bytes_helper = lambda str, length: s.decode('utf-8') + bytearray(length-len(str))` so I can write:
```python
assert c.translator.decode_event(logs.topics, logs.data) ==  \
        {'arg1': b'bar\x00', 'arg2': bytes_helper('bar', 29), 'arg3': bytes_helper('foo', 31), '_event_type': b'MyLog'}
```